assignment/impl/src/java/org/sakaiproject/assignment/impl/GradeSheetExporter.java

```diff
@@ -294,7 +294,14 @@ public int compare(final Submitter s1, final Submitter s2) {
 
                             // Create item and fill up if doesn't exist.
                             // find right row
-    
+                            if (params.get("estimate") != null && params.get("estimate").equals("true")) {
```

Comment:
"true".equals() avoid a possible NPE

Reply:
In this situation, there is already a null check, so there is no possible NPE here. Another option is to use `StringUtils.equals(arg1, arg2)`

Reply:
if changed to:
if ("true".equals(params.get("estimate"))) {